tailwind.config.js

```diff
@@ -1,26 +1,29 @@
 module.exports = {
-    purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
-    darkMode: false, // or 'media' or 'class'
-    theme: {
-        extend: {
-            flex: {
-                navigation: '0 0 40px'
-            },
-            boxShadow: {
-                navigation: '8px 4px 8px 0 rgba(0,0,0,0.2)'
-            },
-            colors: {
-                unranked: '#CD7F32',
-                silver: '#C0C0C0',
-                gold: '#FFD700',
-                platinum: '#008B8B',
-                diamond: '#B9F2FF',
-                max: '#B9F2FF'
-            }
-        }
-    },
-    variants: {
-        extend: {},
-    },
-    plugins: [],
+  purge: [
+    './src/**/*.html',
```

Comment:
what is purge

Reply:
Whenever you have a production build, tailwind will automatically drop all css classes that aren't used in an effort to keep the bundle size down. Since this is an electron app and not necessarily a web app it probably doesn't matter but I'm purging the unused css anyways